test/KoreBuild.FunctionalTests/SimpleRepoTests.cs

```diff
@@ -55,5 +56,23 @@ public SimpleRepoTests(ITestOutputHelper output, RepoTestFixture fixture)
             Assert.Same(task, build);
             Assert.NotEqual(0, build.Result);
         }
+
+        [Fact]
+        public async Task DockerSuccessful()
+        {
+            var app = _fixture.CreateTestApp("SimpleRepo");
+            var platform = "jessie";
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
You should check which platform the docker host is using, not which platform is running this test. Windows machines can use Linux containers.

Also, I would be good to make this test skip if docker is not installed.

```
docker version -f "{{ .Server.Os }}"
```